runtime/flang/async.c

```diff
@@ -292,11 +292,11 @@ Fio_asy_read(struct asy *asy, void *adr, long len)
   tn = asy->outstanding_transactions;
   asy->overlap[tn].Internal = 0;
   asy->overlap[tn].InternalHigh = 0;
-  asy->overlap[tn].u.Pointer = 0;
+  asy->overlap[tn].Pointer = 0;
   /* Load asy->off into OffsetHigh/Offset */
   converter.offset = asy->atd[tn].off;
-  asy->overlap[tn].u.s.Offset = converter.wOffset;
-  asy->overlap[tn].u.s.OffsetHigh = converter.wOffsetHigh;
+  asy->overlap[tn].Offset = converter.wOffset;
```

Comment:
Can your provide a pointer to the OVERLAPPED structure? Is this from a windows header?

Reply:
`.u.s` doesn't work with windows. It throws a compilation error.

Reply:
OK. I didn't see a definition in flang. Assuming it is from a windows header.